packages/core/src/SlotMixin.js

```diff
@@ -57,7 +57,7 @@ export const SlotMixin = dedupeMixin(
             if (!this.$$slot(slotName)) {
               const slotFactory = this.slots[slotName];
               const slotContent = slotFactory();
-              if (slotContent instanceof Element) {
+              if (slotContent instanceof Element || slotContent instanceof HTMLElement) {
```

Comment:
can we add a test for that - seems rather important

Reply:
the issue was in the application code

in short, they had this in their Polymer app which replaced global Element object

```html
<script>
  const { Element } = window.Polymer;
</script>
```

so like we decided a while ago, there is no need to change anything on our side